implementation/src/main/java/io/smallrye/jwt/config/ConfigLogging.java

```diff
@@ -30,4 +30,8 @@
     @LogMessage(level = Logger.Level.DEBUG)
     @Message(id = 3004, value = "mpJwtPublicKey parsed as PEM")
     void publicKeyParsedAsPem();
-}
\ No newline at end of file
+
+    @LogMessage(level = Logger.Level.WARN)
+    @Message(id = 3005, value = "The config %s was replaced by %s. It may be removed in a future version.")
```

Comment:
I propose to rephrase it as `'%s' property is deprecated and will be removed in a future version. Use '%s ' property instead'.

Reply:
Ok.